DOCSP-36929: 1.11.2 version updates (#155)
mongodb/docs-kafka-connector: source/whats-new.txt

* :ref:`Version 1.1 <kafka-connector-whats-new-1.1>`
* :ref:`Version 1.0 <kafka-connector-whats-new-1.0>`

.. _kafka-connector-whats-new-1.11.2:

What's New in 1.11.2
--------------------

> - Fixed a bug in which unsuccessful attempts to retrieve items from a change stream were logged as
>   INFO-level instead of ERROR-level events. To learn more about this fix, see the
>   `KAFKA-396 <https://jira.mongodb.org/browse/KAFKA-396>`__ JIRA issue.

Reviewer: ```suggestion
- Fixed a bug in which unsuccessful attempts to retrieve items from a change stream were logged at the
  ``INFO`` level instead of at ``ERROR`` level. To learn more about this fix, see the
  `KAFKA-396 <https://jira.mongodb.org/browse/KAFKA-396>`__ JIRA issue.
```
